web/server/vue-cli/src/store/modules/base-filter.js

```diff
@@ -13,6 +13,11 @@ const getters = {
     return state.reportFilter;
   },
   getCmpData(state) {
```

Comment:
Where is a usage of this function?

Reply:
Here: https://github.com/Ericsson/codechecker/blob/ff8d468fb318dd32557a82998a2520c1c0ed553d/web/server/vue-cli/src/views/Reports.vue#L291